props/saber_fett263_buttons.h

```diff
@@ -2117,14 +2105,31 @@ SaberFett263Buttons() : PropBase() {}
       sound_library_.SayConfirm();
       break;
 #endif
+    case MENU_STYLE_SUB:
+      switch (menu_sub_pos_) {
+        case EDIT_STYLE_SELECT:
+	  menu_type_ = MENU_STYLE;
```

Comment:
indentation looks weird here, not sure if it's github or the code.
It could be that you have tab characters in the code, which I recommend avoiding in favor of spaces.